services/data-info/src/data_info/routes/avus.clj

```diff
@@ -13,25 +14,30 @@
     (GET* "/avus" [:as {uri :uri}]
       :query [{:keys [user]} StandardUserQueryParams]
       :return AVUGetResult
+      :middlewares [wrap-metadata-base-url]
```

Comment:
This isn't always true but metadata will probably be relevant for everything here, so I think I'd say put this right into the `context*` instead of each of the routes. But I'd be receptive to other arguments.